src/database/Databases.ts

```diff
@@ -125,6 +120,18 @@ export default class MongoDatabase {
     };
   }
 
+  createRedisConnection(): Promise<void> {
+    return new Promise((resolve, reject) => {
+      try {
+        this.redisClient = new Redis({ path: process.env.REDIS_PATH as string });
+        resolve();
+      } catch (err) {
+        console.log(`[REDIS] Error connecting to redis ${err}`);
+        reject(err);
+      }
+    });
+  }
```

Comment:
Not sure why you opted for a promise in the first place

```suggestion
  createRedisConnection(): void {
    try {
      this.redisClient = new Redis({ path: process.env.REDIS_PATH as string });
    } catch (err) {
      console.log(`[REDIS] Error connecting to redis ${err}`);
      throw err;
    }
  }
```